extensions/json-schema/rest/src/main/java/org/apache/unomi/schema/rest/JsonSchemaEndPoint.java

```diff
@@ -85,14 +100,13 @@ public Response save(String jsonSchema) {
     }
 
     /**
-     * Deletes a JSON schema.
-     * The id is a Base64 id as the id have is basically an URL
+     * Deletes a JSON schema by it's id.
      *
-     * @param base64JsonSchemaId the identifier of the JSON schema that we want to delete
+     * @param id the identifier of the JSON schema that we want to delete
      */
-    @DELETE
-    @Path("/{base64JsonSchemaId}")
-    public void remove(@PathParam("base64JsonSchemaId") String base64JsonSchemaId) {
-        schemaService.deleteSchema(new String(Base64.getDecoder().decode(base64JsonSchemaId)));
+    @POST
+    @Path("/delete")
+    public void remove(String id) {
+        schemaService.deleteSchema(id);
     }
```

Comment:
Could be great to have a response here, in case the id doest exist for exemple, or even if the id is matching a predefined schema, in this case the schemaService.deleteSchema is returning false. 
So it could be great to reflect the status of the service return in the endPoint.